gdc_ng_models/models/entity_set.py

```diff
@@ -23,6 +23,23 @@
 from sqlalchemy.dialects import postgresql
 from sqlalchemy.ext import declarative
 from gdc_ng_models.models import accessed, audit
+import enum
+
+
+@enum.unique
+class SetType(enum.Enum):
+    ephemeral = 1
+    frozen = 2
+    mutable = 3
+
+
+@enum.unique
+class EntityType(enum.Enum):
+    case = 1
+    file = 2
+    gene = 3
+    ssm = 4
```

Comment:
nit: all caps for each enum value?

Although I did not find it in google's style guide, afaik, it's been common practice.

See all examples in docs:
https://docs.python.org/3/library/enum.html

weirdly enough, pep 435 shows all examples in lowercase 🤦🏻 
https://peps.python.org/pep-0435/

Reply:
nm ... seems like we need to match existing values.